packages/eas-cli/src/user/SessionManager.ts

```diff
@@ -194,6 +210,16 @@ export default class SessionManager {
     }
   }
 
+  private async ssoLoginAsync(): Promise<void> {
+    const { sessionSecret, id, username } = await fetchSessionSecretAndSsoUserAsync();
+    await this.setSessionAsync({
+      sessionSecret,
+      userId: id,
+      username,
+      currentConnection: 'Username-Password-Authentication',
```

Comment:
This value seems incorrect. Not sure what it should be though.

Reply:
Changed to Browser-Flow-Authentication

Reply:
This change also needs to be made in expo cli